// useritem/dist/useritem.es.js
import De from "react";
var oe = { exports: {} }, U = {};
/**
 * @license React
 * react-jsx-runtime.production.min.js
 *
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
var ke;
function vr() {
  if (ke)
    return U;
  ke = 1;
  var F = De, x = Symbol.for("react.element"), I = Symbol.for("react.fragment"), R = Object.prototype.hasOwnProperty, T = F.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ReactCurrentOwner, w = { key: !0, ref: !0, __self: !0, __source: !0 };
  function C(y, d, b) {
    var f, p = {}, _ = null, P = null;
    b !== void 0 && (_ = "" + b), d.key !== void 0 && (_ = "" + d.key), d.ref !== void 0 && (P = d.ref);
    for (f in d)
      R.call(d, f) && !w.hasOwnProperty(f) && (p[f] = d[f]);
    if (y && y.defaultProps)
      for (f in d = y.defaultProps, d)
        p[f] === void 0 && (p[f] = d[f]);
    return { $$typeof: x, type: y, key: _, ref: P, props: p, _owner: T.current };
  }
  return U.Fragment = I, U.jsx = C, U.jsxs = C, U;
}
var V = {};
/**
 * @license React
 * react-jsx-runtime.development.js
 *
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
var Ae;
function pr() {
  return Ae || (Ae = 1, process.env.NODE_ENV !== "production" && function() {
    var F = De, x = Symbol.for("react.element"), I = Symbol.for("react.portal"), R = Symbol.for("react.fragment"), T = Symbol.for("react.strict_mode"), w = Symbol.for("react.profiler"), C = Symbol.for("react.provider"), y = Symbol.for("react.context"), d = Symbol.for("react.forward_ref"), b = Symbol.for("react.suspense"), f = Symbol.for("react.suspense_list"), p = Symbol.for("react.memo"), _ = Symbol.for("react.lazy"), P = Symbol.for("react.offscreen"), M = Symbol.iterator, k = "@@iterator";
    function G(e) {
      if (e === null || typeof e != "object")
        return null;
      var r = M && e[M] || e[k];
      return typeof r == "function" ? r : null;
    }
    var S = F.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
    function c(e) {
      {
        for (var r = arguments.length, t = new Array(r > 1 ? r - 1 : 0), n = 1; n < r; n++)
          t[n - 1] = arguments[n];
        W("error", e, t);
      }
    }
    function W(e, r, t) {
      {
        var n = S.ReactDebugCurrentFrame, o = n.getStackAddendum();
        o !== "" && (r += "%s", t = t.concat([o]));
        var l = t.map(function(i) {
          return String(i);
        });
        l.unshift("Warning: " + r), Function.prototype.apply.call(console[e], console, l);
      }
    }
    var N = !1, Fe = !1, Ie = !1, We = !1, $e = !1, le;
    le = Symbol.for("react.module.reference");
    function Ye(e) {
      return !!(typeof e == "string" || typeof e == "function" || e === R || e === w || $e || e === T || e === b || e === f || We || e === P || N || Fe || Ie || typeof e == "object" && e !== null && (e.$$typeof === _ || e.$$typeof === p || e.$$typeof === C || e.$$typeof === y || e.$$typeof === d || // This needs to include all possible module reference object
      // types supported by any Flight configuration anywhere since
      // we don't know which Flight build this will end up being used
      // with.
      e.$$typeof === le || e.getModuleId !== void 0));
    }
    function Le(e, r, t) {
      var n = e.displayName;
      if (n)
        return n;
      var o = r.displayName || r.name || "";
      return o !== "" ? t + "(" + o + ")" : t;
    }
    function ue(e) {
      return e.displayName || "Context";
    }
    function m(e) {
      if (e == null)
        return null;
      if (typeof e.tag == "number" && c("Received an unexpected object in getComponentNameFromType(). This is likely a bug in React. Please file an issue."), typeof e == "function")
        return e.displayName || e.name || null;
      if (typeof e == "string")
        return e;
      switch (e) {
        case R:
          return "Fragment";
        case I:
          return "Portal";
        case w:
          return "Profiler";
        case T:
          return "StrictMode";
        case b:
          return "Suspense";
        case f:
          return "SuspenseList";
      }
      if (typeof e == "object")
        switch (e.$$typeof) {
          case y:
            var r = e;
            return ue(r) + ".Consumer";
          case C:
            var t = e;
            return ue(t._context) + ".Provider";
          case d:
            return Le(e, e.render, "ForwardRef");
          case p:
            var n = e.displayName || null;
            return n !== null ? n : m(e.type) || "Memo";
          case _: {
            var o = e, l = o._payload, i = o._init;
            try {
              return m(i(l));
            } catch {
              return null;
            }
          }
        }
      return null;
    }
    var j = Object.assign, $ = 0, se, fe, ce, de, ve, pe, he;
    function ge() {
    }
    ge.__reactDisabledLog = !0;
    function Ue() {
      {
        if ($ === 0) {
          se = console.log, fe = console.info, ce = console.warn, de = console.error, ve = console.group, pe = console.groupCollapsed, he = console.groupEnd;
          var e = {
            configurable: !0,
            enumerable: !0,
            value: ge,
            writable: !0
          };
          Object.defineProperties(console, {
            info: e,
            log: e,
            warn: e,
            error: e,
            group: e,
            groupCollapsed: e,
            groupEnd: e
          });
        }
        $++;
      }
    }
    function Ve() {
      {
        if ($--, $ === 0) {
          var e = {
            configurable: !0,
            enumerable: !0,
            writable: !0
          };
          Object.defineProperties(console, {
            log: j({}, e, {
              value: se
            }),
            info: j({}, e, {
              value: fe
            }),
            warn: j({}, e, {
              value: ce
            }),
            error: j({}, e, {
              value: de
            }),
            group: j({}, e, {
              value: ve
            }),
            groupCollapsed: j({}, e, {
              value: pe
            }),
            groupEnd: j({}, e, {
              value: he
            })
          });
        }
        $ < 0 && c("disabledDepth fell below zero. This is a bug in React. Please file an issue.");
      }
    }
    var q = S.ReactCurrentDispatcher, X;
    function z(e, r, t) {
      {
        if (X === void 0)
          try {
            throw Error();
          } catch (o) {
            var n = o.stack.trim().match(/\n( *(at )?)/);
            X = n && n[1] || "";
          }
        return `
` + X + e;
      }
    }
    var Z = !1, B;
    {
      var Me = typeof WeakMap == "function" ? WeakMap : Map;
      B = new Me();
    }
    function be(e, r) {
      if (!e || Z)
        return "";
      {
        var t = B.get(e);
        if (t !== void 0)
          return t;
      }
      var n;
      Z = !0;
      var o = Error.prepareStackTrace;
      Error.prepareStackTrace = void 0;
      var l;
      l = q.current, q.current = null, Ue();
      try {
        if (r) {
          var i = function() {
            throw Error();
          };
          if (Object.defineProperty(i.prototype, "props", {
            set: function() {
              throw Error();
            }
          }), typeof Reflect == "object" && Reflect.construct) {
            try {
              Reflect.construct(i, []);
            } catch (h) {
              n = h;
            }
            Reflect.construct(e, [], i);
          } else {
            try {
              i.call();
            } catch (h) {
              n = h;
            }
            e.call(i.prototype);
          }
        } else {
          try {
            throw Error();
          } catch (h) {
            n = h;
          }
          e();
        }
      } catch (h) {
        if (h && n && typeof h.stack == "string") {
          for (var a = h.stack.split(`
`), v = n.stack.split(`
`), u = a.length - 1, s = v.length - 1; u >= 1 && s >= 0 && a[u] !== v[s]; )
            s--;
          for (; u >= 1 && s >= 0; u--, s--)
            if (a[u] !== v[s]) {
              if (u !== 1 || s !== 1)
                do
                  if (u--, s--, s < 0 || a[u] !== v[s]) {
                    var g = `
` + a[u].replace(" at new ", " at ");
                    return e.displayName && g.includes("<anonymous>") && (g = g.replace("<anonymous>", e.displayName)), typeof e == "function" && B.set(e, g), g;
                  }
                while (u >= 1 && s >= 0);
              break;
            }
        }
      } finally {
        Z = !1, q.current = l, Ve(), Error.prepareStackTrace = o;
      }
      var D = e ? e.displayName || e.name : "", O = D ? z(D) : "";
      return typeof e == "function" && B.set(e, O), O;
    }
    function Ne(e, r, t) {
      return be(e, !1);
    }
    function ze(e) {
      var r = e.prototype;
      return !!(r && r.isReactComponent);
    }
    function J(e, r, t) {
      if (e == null)
        return "";
      if (typeof e == "function")
        return be(e, ze(e));
      if (typeof e == "string")
        return z(e);
      switch (e) {
        case b:
          return z("Suspense");
        case f:
          return z("SuspenseList");
      }
      if (typeof e == "object")
        switch (e.$$typeof) {
          case d:
            return Ne(e.render);
          case p:
            return J(e.type, r, t);
          case _: {
            var n = e, o = n._payload, l = n._init;
            try {
              return J(l(o), r, t);
            } catch {
            }
          }
        }
      return "";
    }
    var Y = Object.prototype.hasOwnProperty, ye = {}, me = S.ReactDebugCurrentFrame;
    function H(e) {
      if (e) {
        var r = e._owner, t = J(e.type, e._source, r ? r.type : null);
        me.setExtraStackFrame(t);
      } else
        me.setExtraStackFrame(null);
    }
    function Be(e, r, t, n, o) {
      {
        var l = Function.call.bind(Y);
        for (var i in e)
          if (l(e, i)) {
            var a = void 0;
            try {
              if (typeof e[i] != "function") {
                var v = Error((n || "React class") + ": " + t + " type `" + i + "` is invalid; it must be a function, usually from the `prop-types` package, but received `" + typeof e[i] + "`.This often happens because of typos such as `PropTypes.function` instead of `PropTypes.func`.");
                throw v.name = "Invariant Violation", v;
              }
              a = e[i](r, i, n, t, null, "SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED");
            } catch (u) {
              a = u;
            }
            a && !(a instanceof Error) && (H(o), c("%s: type specification of %s `%s` is invalid; the type checker function must return `null` or an `Error` but returned a %s. You may have forgotten to pass an argument to the type checker creator (arrayOf, instanceOf, objectOf, oneOf, oneOfType, and shape all require an argument).", n || "React class", t, i, typeof a), H(null)), a instanceof Error && !(a.message in ye) && (ye[a.message] = !0, H(o), c("Failed %s type: %s", t, a.message), H(null));
          }
      }
    }
    var Je = Array.isArray;
    function Q(e) {
      return Je(e);
    }
    function He(e) {
      {
        var r = typeof Symbol == "function" && Symbol.toStringTag, t = r && e[Symbol.toStringTag] || e.constructor.name || "Object";
        return t;
      }
    }
    function Ke(e) {
      try {
        return Ee(e), !1;
      } catch {
        return !0;
      }
    }
    function Ee(e) {
      return "" + e;
    }
    function Re(e) {
      if (Ke(e))
        return c("The provided key is an unsupported type %s. This value must be coerced to a string before before using it here.", He(e)), Ee(e);
    }
    var L = S.ReactCurrentOwner, Ge = {
      key: !0,
      ref: !0,
      __self: !0,
      __source: !0
    }, _e, xe, ee;
    ee = {};
    function qe(e) {
      if (Y.call(e, "ref")) {
        var r = Object.getOwnPropertyDescriptor(e, "ref").get;
        if (r && r.isReactWarning)
          return !1;
      }
      return e.ref !== void 0;
    }
    function Xe(e) {
      if (Y.call(e, "key")) {
        var r = Object.getOwnPropertyDescriptor(e, "key").get;
        if (r && r.isReactWarning)
          return !1;
      }
      return e.key !== void 0;
    }
    function Ze(e, r) {
      if (typeof e.ref == "string" && L.current && r && L.current.stateNode !== r) {
        var t = m(L.current.type);
        ee[t] || (c('Component "%s" contains the string ref "%s". Support for string refs will be removed in a future major release. This case cannot be automatically converted to an arrow function. We ask you to manually fix this case by using useRef() or createRef() instead. Learn more about using refs safely here: https://reactjs.org/link/strict-mode-string-ref', m(L.current.type), e.ref), ee[t] = !0);
      }
    }
    function Qe(e, r) {
      {
        var t = function() {
          _e || (_e = !0, c("%s: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://reactjs.org/link/special-props)", r));
        };
        t.isReactWarning = !0, Object.defineProperty(e, "key", {
          get: t,
          configurable: !0
        });
      }
    }
    function er(e, r) {
      {
        var t = function() {
          xe || (xe = !0, c("%s: `ref` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://reactjs.org/link/special-props)", r));
        };
        t.isReactWarning = !0, Object.defineProperty(e, "ref", {
          get: t,
          configurable: !0
        });
      }
    }
    var rr = function(e, r, t, n, o, l, i) {
      var a = {
        // This tag allows us to uniquely identify this as a React Element
        $$typeof: x,
        // Built-in properties that belong on the element
        type: e,
        key: r,
        ref: t,
        props: i,
        // Record the component responsible for creating this element.
        _owner: l
      };
      return a._store = {}, Object.defineProperty(a._store, "validated", {
        configurable: !1,
        enumerable: !1,
        writable: !0,
        value: !1
      }), Object.defineProperty(a, "_self", {
        configurable: !1,
        enumerable: !1,
        writable: !1,
        value: n
      }), Object.defineProperty(a, "_source", {
        configurable: !1,
        enumerable: !1,
        writable: !1,
        value: o
      }), Object.freeze && (Object.freeze(a.props), Object.freeze(a)), a;
    };
    function tr(e, r, t, n, o) {
      {
        var l, i = {}, a = null, v = null;
        t !== void 0 && (Re(t), a = "" + t), Xe(r) && (Re(r.key), a = "" + r.key), qe(r) && (v = r.ref, Ze(r, o));
        for (l in r)
          Y.call(r, l) && !Ge.hasOwnProperty(l) && (i[l] = r[l]);
        if (e && e.defaultProps) {
          var u = e.defaultProps;
          for (l in u)
            i[l] === void 0 && (i[l] = u[l]);
        }
        if (a || v) {
          var s = typeof e == "function" ? e.displayName || e.name || "Unknown" : e;
          a && Qe(i, s), v && er(i, s);
        }
        return rr(e, a, v, o, n, L.current, i);
      }
    }
    var re = S.ReactCurrentOwner, we = S.ReactDebugCurrentFrame;
    function A(e) {
      if (e) {
        var r = e._owner, t = J(e.type, e._source, r ? r.type : null);
        we.setExtraStackFrame(t);
      } else
        we.setExtraStackFrame(null);
    }
    var te;
    te = !1;
    function ne(e) {
      return typeof e == "object" && e !== null && e.$$typeof === x;
    }
    function Se() {
      {
        if (re.current) {
          var e = m(re.current.type);
          if (e)
            return `

Check the render method of \`` + e + "`.";
        }
        return "";
      }
    }
    function nr(e) {
      return "";
    }
    var Te = {};
    function ar(e) {
      {
        var r = Se();
        if (!r) {
          var t = typeof e == "string" ? e : e.displayName || e.name;
          t && (r = `

Check the top-level render call using <` + t + ">.");
        }
        return r;
      }
    }
    function Ce(e, r) {
      {
        if (!e._store || e._store.validated || e.key != null)
          return;
        e._store.validated = !0;
        var t = ar(r);
        if (Te[t])
          return;
        Te[t] = !0;
        var n = "";
        e && e._owner && e._owner !== re.current && (n = " It was passed a child from " + m(e._owner.type) + "."), A(e), c('Each child in a list should have a unique "key" prop.%s%s See https://reactjs.org/link/warning-keys for more information.', t, n), A(null);
      }
    }
    function je(e, r) {
      {
        if (typeof e != "object")
          return;
        if (Q(e))
          for (var t = 0; t < e.length; t++) {
            var n = e[t];
            ne(n) && Ce(n, r);
          }
        else if (ne(e))
          e._store && (e._store.validated = !0);
        else if (e) {
          var o = G(e);
          if (typeof o == "function" && o !== e.entries)
            for (var l = o.call(e), i; !(i = l.next()).done; )
              ne(i.value) && Ce(i.value, r);
        }
      }
    }
    function ir(e) {
      {
        var r = e.type;
        if (r == null || typeof r == "string")
          return;
        var t;
        if (typeof r == "function")
          t = r.propTypes;
        else if (typeof r == "object" && (r.$$typeof === d || // Note: Memo only checks outer props here.
        // Inner props are checked in the reconciler.
        r.$$typeof === p))
          t = r.propTypes;
        else
          return;
        if (t) {
          var n = m(r);
          Be(t, e.props, "prop", n, e);
        } else if (r.PropTypes !== void 0 && !te) {
          te = !0;
          var o = m(r);
          c("Component %s declared `PropTypes` instead of `propTypes`. Did you misspell the property assignment?", o || "Unknown");
        }
        typeof r.getDefaultProps == "function" && !r.getDefaultProps.isReactClassApproved && c("getDefaultProps is only used on classic React.createClass definitions. Use a static property named `defaultProps` instead.");
      }
    }
    function or(e) {
      {
        for (var r = Object.keys(e.props), t = 0; t < r.length; t++) {
          var n = r[t];
          if (n !== "children" && n !== "key") {
            A(e), c("Invalid prop `%s` supplied to `React.Fragment`. React.Fragment can only have `key` and `children` props.", n), A(null);
            break;
          }
        }
        e.ref !== null && (A(e), c("Invalid attribute `ref` supplied to `React.Fragment`."), A(null));
      }
    }
    var Oe = {};
    function Pe(e, r, t, n, o, l) {
      {
        var i = Ye(e);
        if (!i) {
          var a = "";
          (e === void 0 || typeof e == "object" && e !== null && Object.keys(e).length === 0) && (a += " You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.");
          var v = nr();
          v ? a += v : a += Se();
          var u;
          e === null ? u = "null" : Q(e) ? u = "array" : e !== void 0 && e.$$typeof === x ? (u = "<" + (m(e.type) || "Unknown") + " />", a = " Did you accidentally export a JSX literal instead of a component?") : u = typeof e, c("React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: %s.%s", u, a);
        }
        var s = tr(e, r, t, o, l);
        if (s == null)
          return s;
        if (i) {
          var g = r.children;
          if (g !== void 0)
            if (n)
              if (Q(g)) {
                for (var D = 0; D < g.length; D++)
                  je(g[D], e);
                Object.freeze && Object.freeze(g);
              } else
                c("React.jsx: Static children should always be an array. You are likely explicitly calling React.jsxs or React.jsxDEV. Use the Babel transform instead.");
            else
              je(g, e);
        }
        if (Y.call(r, "key")) {
          var O = m(e), h = Object.keys(r).filter(function(dr) {
            return dr !== "key";
          }), ae = h.length > 0 ? "{key: someKey, " + h.join(": ..., ") + ": ...}" : "{key: someKey}";
          if (!Oe[O + ae]) {
            var cr = h.length > 0 ? "{" + h.join(": ..., ") + ": ...}" : "{}";
            c(`A props object containing a "key" prop is being spread into JSX:
  let props = %s;
  <%s {...props} />
React keys must be passed directly to JSX without using spread:
  let props = %s;
  <%s key={someKey} {...props} />`, ae, O, cr, O), Oe[O + ae] = !0;
          }
        }
        return e === R ? or(s) : ir(s), s;
      }
    }
    function lr(e, r, t) {
      return Pe(e, r, t, !0);
    }
    function ur(e, r, t) {
      return Pe(e, r, t, !1);
    }
    var sr = ur, fr = lr;
    V.Fragment = R, V.jsx = sr, V.jsxs = fr;
  }()), V;
}
process.env.NODE_ENV === "production" ? oe.exports = vr() : oe.exports = pr();
var E = oe.exports;
const K = 42, ie = 12;
function gr({
  avatar: F = !0,
  avatarUrl: x,
  backgroundColor: I = "#03b66e",
  border: R = !0,
  description: T = "Youtuber",
  disabled: w = !1,
  width: C = 275,
  onClick: y,
  online: d = !1,
  onlyAvatar: b = !1,
  reverse: f = !1,
  squared: p = !1,
  status: _ = !0,
  shadow: P = !1,
  style: M = {},
  name: k = "Guillaume Duhan",
  verified: G = !0
}) {
  const S = (W) => {
    y && y(W), alert("Hello there!");
  }, c = () => {
    if (!x)
      return k ? k.split(" ").map((W) => {
        var N;
        return (N = W[0]) == null ? void 0 : N.toUpperCase();
      }).join("") : "AA";
  };
  return /* @__PURE__ */ E.jsxs("div", { style: {
    display: "flex",
    alignItems: "center",
    padding: b ? "" : "10px 12px",
    border: R && !b ? "1px solid #ccc" : "",
    borderRadius: p ? "" : "8px",
    flexDirection: f ? "row-reverse" : "row",
    boxShadow: P && R ? "-1px 0px 16px 0px rgba(0,0,0,.05)" : "",
    width: `${b ? "fit-content" : `${C}px`}`,
    maxWidth: "100%",
    gap: "8px",
    backgroundColor: "transparent",
    fontFamily: "inherit, sans-serif",
    color: "black",
    lineHeight: "1.2",
    outline: "none",
    cursor: w ? "none" : "pointer",
    position: "relative",
    pointerEvents: w ? "none" : "all",
    userSelect: "none",
    opacity: w ? 0.5 : 1,
    ...M
  }, onClick: S, children: [
    F && /* @__PURE__ */ E.jsxs("div", { style: { position: "relative" }, children: [
      /* @__PURE__ */ E.jsx("div", { style: {
        overflow: "hidden",
        color: "white",
        display: "flex",
        alignItems: "center",
        justifyContent: "center",
        fontWeight: 800,
        fontSize: "16px",
        outline: "none",
        maxWidth: K,
        minWidth: K,
        maxHeight: K,
        minHeight: K,
        backgroundColor: I || "#ccc",
        backgroundImage: `url(${x})`,
        backgroundSize: "cover",
        borderRadius: p ? "" : "100px"
      }, children: c() }),
      _ && /* @__PURE__ */ E.jsx("div", { style: {
        width: ie,
        height: ie,
        backgroundColor: d ? "limegreen" : "#ccc",
        borderRadius: ie,
        border: "2px solid white",
        position: "absolute",
        bottom: p ? "-6px" : 0,
        right: p ? "-6px" : 0
      } })
    ] }),
    !b && /* @__PURE__ */ E.jsxs("div", { style: {
      textAlign: f ? "right" : "left",
      flexDirection: f ? "row-reverse" : "row",
      width: "calc(80%)"
    }, children: [
      k && /* @__PURE__ */ E.jsxs("div", { style: {
        display: "flex",
        lineHeight: 1.42,
        gap: "4px",
        alignItems: "center",
        justifyContent: f ? "end" : "start"
      }, children: [
        /* @__PURE__ */ E.jsx("div", { style: {
          color: "black",
          fontWeight: 800,
          fontSize: "16px",
          whiteSpace: "nowrap",
          overflow: "hidden",
          textOverflow: "ellipsis"
        }, children: k }),
        G && /* @__PURE__ */ E.jsx("svg", { xmlns: "http://www.w3.org/2000/svg", width: "1.3em", height: "1.3em", viewBox: "0 0 24 24", children: /* @__PURE__ */ E.jsx("path", { fill: "currentColor", d: "m8.6 22.5l-1.9-3.2l-3.6-.8l.35-3.7L1 12l2.45-2.8l-.35-3.7l3.6-.8l1.9-3.2L12 2.95l3.4-1.45l1.9 3.2l3.6.8l-.35 3.7L23 12l-2.45 2.8l.35 3.7l-3.6.8l-1.9 3.2l-3.4-1.45zm2.35-6.95L16.6 9.9l-1.4-1.45l-4.25 4.25l-2.15-2.1L7.4 12z" }) })
      ] }),
      T && /* @__PURE__ */ E.jsx("div", { style: {
        color: "#AAA",
        fontWeight: 500,
        fontSize: "16px",
        whiteSpace: "nowrap",
        overflow: "hidden",
        textOverflow: "ellipsis"
      }, children: T })
    ] })
  ] });
}
export {
  gr as default
};
